src/main/java/com/st/lms/exception/ApiError.java

```diff
@@ -0,0 +1,40 @@
+package com.st.lms.exception;
+
+public class ApiError {
+
+	private int statusCode;

```

Comment:
To repeat my comment on STOzaki/ilLmsSpring#1, why not use `HttpStatus` (or whatever the Spring-provided `enum` is called) here?